Branch-SDK/src/io/branch/referral/PrefHelper.java

```diff
@@ -1024,6 +1032,18 @@ public void disableSmartSession() {
         BNC_Smart_Session = false;
     }
 
+    public void setRequestMetadata(@NonNull String key, Object value) {
```

Comment:
Since retadata Object is added and retrieved as  JSON Object internally is there any reason for specifying param type as `Object`